x-pack/test/security_solution_cypress/cypress/e2e/explore/dashboards/entity_analytics.cy.ts

```diff
@@ -544,7 +544,8 @@ describe('Entity Analytics Dashboard', { tags: ['@ess', '@serverless'] }, () =>
     });
   });
 
-  describe('With anomalies data', () => {
+  // Skipping to unblock: https://github.com/elastic/kibana/pull/168389
+  describe.skip('With anomalies data', () => {
```

Comment:
I can reproduce this when running locally with Network throttling.

Reply:
Could you please check if the error you had was the same as https://github.com/elastic/kibana/pull/168709, if so, can you please try unskipping the test?

Reply:
I removed skip, it is not longer needed now that test has been fixed